setup.py

```diff
@@ -34,14 +34,14 @@
     long_description=long_description,
     url='https://github.com/fedspendingtransparency/data-act-broker-backend.git',
     author='US Treasury',
-    author_email='na@na.com',
+    author_email='databroker@fiscal.treasury.gov',
     license='CC0 1.0 Universal (CC0 1.0) Public Domain Dedication',
     classifiers=[
-        'Development Status :: 3 - Alpha',
+        'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
         'Topic :: Software Development :: Build Tools',
         'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication',
-        'Programming Language :: Python :: 2.7'
+        'Programming Language :: Python :: 3.4'
```

Comment:
I'm not really sure what this section is doing, but does this limit us to 3.4 in any way?  I run locally on 3.5, but I never actually run this setup.py file so I'm not sure if that would be a problem.